examples/automm/Kaggle-Pawpularity/README.md

```diff
@@ -0,0 +1,103 @@
+
+### 2.1 Build the AutoMMPredictor
+
+You can build the predictor as follow.
+
+    predictor = AutoMMPredictor(
+	    label="Pawpularity", 
+	    problem_type="regression", 
+	    eval_metric="rmse", 
+	    path=save_path,  
+	    verbosity=4, 
+	)
+
+ - `label`indicates the target value in training data.
+ - `problem_type`indicates the type of the problem. That can be "Multiclass", "Binary" or "Regression".
+ - `eval_metric`indicates the evaluation index of the model which is always the evaluation of the competition.
+ - `path`indicates the path to save AutoMMPredictor models.
+ - `verbosity`controls how much information is printed.
+
+### 2.2 Train the AutoMMPredictor
+
+ Then, you can train the AutoMMPredictor with `.fit()`.
+ 
+
+    predictor.fit(  
+	    train_data=training_df,
+	    tuning_data=valid_df,  
+	    save_path=save_path,  
+	    hyperparameters={
+		    "model.names": "['timm_image']",
+		    "model.timm_image.checkpoint_name": "swin_large_patch4_window7_224",
+		    "model.timm_image.train_transform_types": "['resize_shorter_side','center_crop','randaug']",
+		    "data.categorical.convert_to_text": "False",
+		    "env.per_gpu_batch_size": "16",
+		    "env.per_gpu_batch_size_evaluation": "32",
+		    "env.precision": "32",
+		    "optimization.learning_rate": "2e-5",
+		    "optimization.weight_decay": "0",
+		    "optimization.lr_decay": "1",
+		    "optimization.max_epochs": "5",
+		    "optimization.warmup_steps": "0",
+		    "optimization.loss_function": "bcewithlogitsloss",
+		},
+		seed=1,
+	)
+
+ - `train_data`is the data used for training.
+ - `tuning_data`is the data for validation. If it is empty, the tuning data will be split from training data automatically.
+ - `save_path`indicates the specific path for model saving in a fit process.
+ - `hyperparameters`is a Dict which will override the default configs in the training. The configs contain five different types.  
+ -- `model`contains the parameters which control the models used in the predictor. You can select the model you need and adjust the details. Default is selecting the models determined by dataset automatically.
+ --`data`contains the configs of transforms for different types of data. 
+ --`env`contains the configs of the training environment.
+ --`optimization`contains the configs in the optimization process, including but not limit to max training epochs, learning rate and warm up.
+ - `seed`determines the random seed.
+
+## 3. Prediction in Kaggle Competitions
+
+In kaggle competitions, the predictor you trained will be used to predict the test dataset. You should upload the AutoMMPredictor standalone save files as a dataset to kaggle and put it in the input data sources of your prediction code. 
```

Comment:
Next, let's upload the predictor to Kaggle and use it to generate predictions on the test set. You should upload the AutoMMPredictor standalone save files as a dataset to kaggle and put it in the input data sources of your prediction code.